docs/source/user_guide/fundamentals/public_api.rst

```diff
@@ -0,0 +1,74 @@
+----------
+Public API
+----------
+
+.. module:: pybamm
+    :noindex:
+
+PyBaMM is a Python package for mathematical modelling and simulation of battery systems. The main classes and functions that are intended to be used by the user are described in this document.
+For a more detailed description of the classes and methods, see the [API documentation](api_docs).
```

Comment:
The API documentation link does not work

Reply:
whoops, thanks @kratman will fix